ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -59,18 +59,37 @@ def _compute_all(self):
 
         user_id = self.env.user.id
         # compute overtime balance
+        #overtime hours
         self.env.cr.execute(
             """
-            SELECT overtime_balanced FROM
-            overtime_balance_report
-            WHERE user_id=%s
+            SELECT sum(unit_amount) FROM
+            ps_time_line
+            WHERE user_id=%s 
+            AND project_id IN 
+            (SELECT id FROM project_project WHERE overtime_hrs = True)
             """,
             (user_id,),
         )
-        overtime_balance = 0
+        overtime_hrs = 0
         for x in self.env.cr.fetchall():
-            overtime_balance += x[0]
-        self.overtime_balance = overtime_balance
+            overtime_hrs += x and x[0] or 0
+
+        # overtime taken
+        self.env.cr.execute(
+            """
+            SELECT sum(unit_amount) FROM
+            ps_time_line
+            WHERE user_id=%s 
+            AND project_id IN 
+            (SELECT id FROM project_project WHERE overtime = True)
+            """,
+            (user_id,),
+        )
+        overtime_taken = 0
+        for x in self.env.cr.fetchall():
+            overtime_taken += x and x[0] or 0
+
+        self.overtime_balance = overtime_hrs - overtime_taken
```

Comment:
Hi @hbrunn @kumarinie 

This again was caused by a request from my side that I did not log in a ticket. When Sushma was setting up her local system with the PS software so that she could replicate the issues we had, we went over several topics such as the required booleans on projects. Based on those booleans, PS time lines are generated that can be seen as overtime lines or overtime taken lines. I did not realise that we had a central calculation for that which is already covering the requirement. Only later I found out that I was myself looking at the amount field in the list view behind the overtime button instead of the quantity field. Therefore wrongly assuming that the calculation must have been wrong.

I believe we shoud revert this change and keep the as-is logic since it appears there is no furthen change needed.